scripts/run_imitation_example.sh

```diff
@@ -3,21 +3,11 @@
 set -e
 set -o pipefail
 
-# We need an older version of setuptools (and pip) to install gym 0.21. Poetry
-# doesn't support specifying the setuptools version used at build time.
-
-# The workaround is:
-
```

Comment:
Finally we can get rid of this :tada: 

Please also remove the part in the `README` that mentions this.